requirements.txt

```diff
@@ -2,4 +2,4 @@
 wheel
 
 # requirements for the CI regression testing scripts
-energyplus-regressions
+energyplus-regressions>=2.1.1
```

Comment:
I'm actually thinking I don't want this in here long term.  The regression workflow can install this.  No need for it when packaging.